tests/keras/backend/backend_test.py

```diff
@@ -12,6 +12,7 @@
 
 
 BACKENDS = []  # Holds a list of all available back-ends
+BACKEND_AND_NP = [K, KNP]
```

Comment:
How about just `WITH_NP`?

Reply:
I'm really bad at naming variables, haha. Maybe `K_AND_KNP` ? That would be the most explicit.

Reply:
You are such a big help :) But I think both `K` and `WITH_NP` are implicit. And `check_single_tensor_operation(op, ..., WITH_NP, ...)` is more natural to read than `K_AND_KNP`.

Reply:
Thanks for your kind words. Let's go with `WITH_NP` then.